README.md

```diff
@@ -11,3 +11,9 @@ To clean up any test output:
 ```bash
 task clean
 ```
+
+To run all linting checks:
+
+```bash
+task lint:check
+```
```

Comment:
Can we add a contributing section like we have in [Spider](https://github.com/y-scope/spider)?